<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagerDlg</class>
 <widget class="QDialog" name="UserManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Img.qrc">
    <normaloff>:/new/prefix1/img/user_group.png</normaloff>:/new/prefix1/img/user_group.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidgetUserInfo">
       <column>
        <property name="text">
         <string>用户名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>密码</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>权限</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonEdit">
           <property name="text">
            <string>编辑</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonDelete">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Img.qrc"/>
 </resources>
 <connections/>
</ui>
